src/containers/LoggedInWrapper/LoggedInWrapper.js

```diff
@@ -2,11 +2,28 @@ import React, { Component } from 'react';
 import { Link, Redirect, withRouter } from 'react-router-dom';
 import { Select, Spin, Icon, Layout, Menu, Typography, Modal, Row, Col } from 'antd';
 import { connect } from 'react-redux';
+import styled from '@emotion/styled';
 
 import { districtsActions, adminActions } from '../../_actions';
 
 import {isSenatorDistrict, displayName, comparator as districtComparator } from '../../_util/district';
 
+const AlertText = styled(Typography.Text)`
+    color: red;
+    font-weight: bold;
+`
+
+const CovidAlert = ({ district }) => {
+    if (district && district.status === "covid_paused") {
+        return (
+            <div style={{padding: "10px"}}>
+                <AlertText>Call notifications are paused for this district due to COVID-19. Update this setting in the Representatives tab.</AlertText>
+            </div>
+        )
+    }
+    return null
+};
```

Comment:
😍 A beautiful functional component! What do think about combining the `padding: 10px` up into the `AlertText` styles. Alternatively, just ditch the custom styles and use ant's `Alert` like: 

```
return (
    <Alert
        message="Call notifications are paused for this district due to COVID-19. Update this setting in the Representatives tab."
        type="error"
    />
)